### DISCUSSION TITLE
[screenshot] Tokyo Night

### DISCUSSION THREAD

**Discussion by mgutz** [qtile/qtile#3622] [Show and tell]
![qtile](https://user-images.githubusercontent.com/32134/172298806-73e19bb1-2394-43d5-ac5d-a4160d46d542.png)

- modified [Tokyo Night VS Code theme](https://github.com/enkia/tokyo-night-vscode-theme)

- [Alacritty theme colors](https://github.com/zatchheems/tokyo-night-alacritty-theme/blob/main/tokyo-night.yaml)

- [picom compositor](https://github.com/yshui/picom) for rounding and opacity

- [wallpaper by Brett Ritchie](https://unsplash.com/photos/S0c_wsCmlXE)

- [zsh prompt by starship](https://starship.rs/)

- Mononoki Nerd Font

I posted this on `r/unixporn` and it got a lot of likes. Figured it should go here too for others interested in ricing their Qtile desktop.

**Comment by m-col** [comment]
Very nice!
